Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -186,6 +231,52 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * Send to trash callback triggering notifications and making app to navigate to parent location
+         *
+         * @method _afterDeleteCallback
+         * @protected
+         * @param {eZ.Location} parentLocation the parent location to which app will navigate to
+         * @param {eZ.Content} content the content to be trashed
+         * @param {Boolean} error
+         */
+        _afterDeleteCallback: function (parentLocation, content, error) {
+            var app = this.get('app'),
+                location = this.get('location'),
+                locationId = location.get('id'),
+                contentName = content.get('name');
+
+            if (error) {
+                this._notify(
+                    'An error occurred when deleting "' + contentName,
+                    'delete-' + locationId,
+                    'error',
+                    0
+                );
+                return;
+            }
+
+            this._notify(
+                '"' + contentName + '" deleted',
+                'delete-' + locationId,
+                'done',
+                5
+            );
+
+            /**
+             * Fired when the content deleted
+             *
+             * @event sentToTrash
+             * @param {eZ.Location} location
+             */
+            this.fire('delete', {location: location});
```

Comment:
that's a bit too generic. I would call it `deleteContent` and I would also add the `content` to the event parameter. (also the `@event` in the doc is wrong)